.pre-commit-config.yaml

```
@@ -35,6 +36,7 @@ repos:
           third_party/SuiteSparse/AMD/.*|
           third_party/identify_stream_usage/.*|
           include/ginkgo/ginkgo.hpp
+          core/base/workspace_aliases.hpp
```

I think it has to be
```suggestion
        examples/external-lib-interfacing/external-lib-interfacing.cpp|
        core/base/workspace_aliases.hpp
```

---

that should also fix the CI.